src/sentry/web/frontend/organization_integration_setup.py

```diff
@@ -37,6 +39,20 @@ def handle(self, request: Request, organization, provider_id) -> HttpResponseBas
                 request=request, organization=organization, provider_key=provider_id
             )
 
+            is_feature_enabled = {}
+            for feature in pipeline.provider.features:
+                feature_flag_name = "organizations:integrations-%s" % feature.value
+                if feature_flag_name in settings.SENTRY_FEATURES:
```

Comment:
@oioki if there is an unregistered feature, we should consider that an enabled feature. I know it's kind of dumb but otherwise some integrations might break.